src/components/TodoLoader/TodoLoader.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  isAdding?: boolean;
+  isDeleting?: boolean;
+  isUpdating?: boolean;
+};
+
+export const TodoLoader: React.FC<Props> = ({
```

Comment:
Vova said that we also add `React.memo` to components that don't get props

Reply:
fixed